### ISSUE TITLE
Quest: Basic Chemistry (QID: 13297)

### ISSUE THREAD

**Issue by Ye-Old-Potato** [YggdrasilWotLK/Yggdrasil-WoW#76]
Quest: Basic Chemistry (QID: 13297)

 The enemies spawn in a strange position that they cannot be targeted (working theory: their Z axis is off || Testing method: Use the q item at the spot, let the stuff spawn, watch them evade, use GM fly to land on top of the pot, and watch yourself able to attack them. The way it is intended to work is that these fellows should show up outside of that pot and climb down to fight ya, you murder em, and when you kill the ghoul that pops up, the quest should complete - The completion does happen but only by actively cheating as a GM. There should be more than 1 wave of ghouls as I remember, like 2-3, you should need to use the quest item anywhere between 2-5 times for it to complete if I remember right. So yeah no this isn't supposed to be an easy quest by any means, but the enemies should still be attackable, which, they currently aren't.

Of note a bit of case history:
https://us.forums.blizzard.com/en/wow/t/basic-chemistry-group-quest-in-ic-impossible-to-do-with-1-person/1360501
^this issue seems to be fixed and a non-issue, but worth knowing in case something here affects future changes.

https://www.youtube.com/watch?v=DGAbcA7kqxQ

**Comment by YggdrasilWotLK** [comment]
Could you please share the .gp coordinates or a nearby creature ID for next such quest where the quest marker doesn't align with the trigger area?

There's multiple reports in one here, I'll try to unbundle them but do let me know if I missed anything:
1. The Z axis seems fine, instead the NPCs are just trapped in the cauldron's LOS. I'm assuming their pathing got broken with an mmaps change at some point. I'll move them out on spawn
2. There are a bunch of waves, this works fine when I kill them while standing on the cauldron or use .damage
3. I used my quest item twice and that was enough, so this seems okay

Tldr; the only issue I can see is the mobs getting stuck out of LOS. I've now made the ghouls leap out as they should and the little oozes spawn in a half circle in front of the cauldron.
